docs/tutorials/compute_for_lambda.rst

```diff
@@ -0,0 +1,314 @@
+Globus Compute for AWS Lambda Developers
+****************************************
+
+This guide is intended for prospective Globus Compute endpoint administrators and users
+who are familiar with AWS Lambda but not with Compute, and aims to explain core
+concepts in Compute in terms of similar concepts in Lambda.
+
+
+Major Differences
+=================
+
+.. list-table::
+    :header-rows: 1
+    :stub-columns: 1
+
+    * -
+      - AWS Lambda
+      - Globus Compute
+    * - Pricing Model
+      - Usage-based billing
+      - Free; :doc:`optional subscription <../subscriptions>`
+    * - Deployment
+      - Everything in the cloud
+      - Hybrid cloud and on-premises
+    * - Execution Environment
+      - Managed by AWS
+      - Managed by endpoint administrators
+    * - Function Definition
+      - Packaged code with handler function
+      - Plain Python functions
+    * - Dependencies
+      - Deployment package, layers
+      - Managed by endpoint administrators
+
+
+Functions & Endpoints
+=====================
+
+In Compute, **functions** are simply Python or Bash source code that users register
+with the service. Unlike Lambda, they are not tied to any particular execution
+context. Instead, when calling a function, users choose an **endpoint** they can
+access to execute their code. Like any Lambda runtime, Compute endpoints are what's
+ultimately responsible for executing a function's code.
+
+To manage Compute endpoints, an endpoint administrator installs `the open source
+package <https://github.com/globus/globus-compute/tree/main/compute_endpoint>`_ on a
+Linux-based machine they want to share with others, then uses the
+``globus-compute-endpoint`` CLI to create endpoints, configure them, and register them
```

Comment:
`s/CLI/command/`

"... then uses the `globus-compute-endpoint` _command_ to create ..."